### ISSUE TITLE
How to set classes weights?

### ISSUE THREAD

**Issue by Bonsen** [bermanmaxim/LovaszSoftmax#11]
In Cross-entropy training,
loss = L.xloss(out, labels, ignore=255)?

**Comment by chenzhutian** [comment]
So how to set the classes weights?

**Comment by bermanmaxim** [comment]
See #13
